src/keysim.js

```diff
@@ -219,6 +221,14 @@ export class Keyboard {
         if (events & KeyEvents.INPUT) {
           const textinputEvent = this.createEventFromKeystroke('textInput', keystroke, target);
           target.dispatchEvent(textinputEvent);
+          const inputEvent = this.createEventFromKeystroke('input', keystroke, target);
+          target.dispatchEvent(inputEvent);
+        }
+      }
+      if (keystroke.keyCode === US_ENGLISH_ACTION_KEYCODE_MAP.BACKSPACE) {
+        if (events & KeyEvents.INPUT) {
+          const inputEvent = this.createEventFromKeystroke('input', keystroke, target);
```

Comment:
This starts to get a little tricky because the `input` event should only fire when the value of the element would change as a result of the key press. So if you backspace in an empty field (or with the cursor at the start) there is no `input` event.